src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Utilities/CastAnalyzer.vb

```diff
@@ -266,13 +277,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
                 End If
 
                 ' If there is an user-defined conversion from the expression to the cast type or the cast
-                ' to the outer type, we need to make sure that the same user-defined conversion will be 
+                ' to the outer type, we need to make sure that the same user-defined conversion will be
                 ' called if the cast is removed.
                 If castToOuterType.IsUserDefined OrElse expressionToCastType.IsUserDefined Then
                     Return (HaveSameUserDefinedConversion(expressionToCastType, expressionToOuterType) OrElse
                             HaveSameUserDefinedConversion(castToOuterType, expressionToOuterType)) AndAlso
-                           (UserDefinedConversionIsAllowed(_castNode) AndAlso
-                            Not expressionToCastType.IsNarrowing)
+                           UserDefinedConversionIsAllowed(_castNode) AndAlso
+                            Not expressionToCastType.IsNarrowing
```

Comment:
this feels more confusing afterwards.

Reply:
Just following Visual Studio suggestion. I can put it back.